src/vector/focus.rs

```diff
@@ -936,6 +971,26 @@ where
         Some(&mut self.get_focus()[target_phys_index])
     }
 
+    pub fn get_many<const N: usize>(
```

Comment:
Maybe call this `get_many_mut` to match the other one? Also, does it need to be `pub`? If so, I think it needs to be documented. If not, maybe you could even make all the internal functions `unsafe` and skip the index-checking. Because internally we're already checking indices in, e.g. `triplet`

Reply:
I skipped the `mut` because the above `get` method also skips it, despite returning a mutable reference. In terms of needing to be pub, it doesn't really need to be, though it may be useful to users. To match the `std` pattern of such APIs, if it stays public, there should likely be an `unsafe` variant that skips the checks, then `triplet` can use it.

Reply:
Makes sense, thanks. I don't have a strong opinion about whether it should be pub or not, just that if it's pub then its documentation should be up to par with the documentation on other pub methods.